test/Domain/LeanCode.DomainModels.MassTransitRelay.Tests/Integration/TestApp.cs

```diff
@@ -103,46 +93,42 @@ public async Task InitializeAsync()
         using var dbContext = scope.Resolve<TestDbContext>();
         await dbContext.Database.EnsureCreatedAsync();
 
-        await bus.StartAsync();
+        await Harness.Start();
     }
 
     public async Task DisposeAsync()
     {
-        await bus.StopAsync();
+        await Harness.Stop();
         await dbConnection.CloseAsync();
         await dbConnection.DisposeAsync();
     }
 }
 
 public class TestMassTransitModule : MassTransitRelayModule
 {
-    public TestMassTransitModule(TypesCatalog eventsCatalog, bool useInbox = true, bool useOutbox = true)
-        : base(eventsCatalog, useInbox, useOutbox) { }
-
     public override void ConfigureMassTransit(IServiceCollection services)
     {
-        services.AddMassTransit(cfg =>
+        services
+            .AddOptions<OutboxDeliveryServiceOptions>()
+            .Configure(opts => opts.QueryDelay = TimeSpan.FromSeconds(1));
+
+        services.AddMassTransitTestHarness(cfg =>
         {
-            cfg.AddConsumers(typeof(TestApp).Assembly);
+            cfg.AddConsumersWithDefaultConfiguration(
+                new[] { typeof(TestApp).Assembly },
+                typeof(DefaultConsumerDefinition<>)
+            );
+
+            cfg.AddEntityFrameworkOutbox<TestDbContext>(outboxCfg =>
+            {
+                outboxCfg.UseSqlite();
+                outboxCfg.UseBusOutbox();
+            });
+
             cfg.UsingInMemory(
                 (ctx, busCfg) =>
                 {
-                    var queueName = Assembly.GetEntryAssembly()!.GetName().Name;
-
-                    busCfg.ReceiveEndpoint(
-                        queueName,
-                        rcv =>
-                        {
-                            rcv.UseLogsCorrelation();
-                            rcv.UseRetry(retryConfig => retryConfig.Immediate(5));
-                            rcv.UseConsumedMessagesFiltering(ctx);
-                            rcv.StoreAndPublishDomainEvents(ctx);
-
-                            rcv.ConfigureConsumers(ctx);
-                            rcv.ConnectReceiveEndpointObservers(ctx);
-                        }
-                    );
-
+                    busCfg.ConfigureEndpoints(ctx);
                     busCfg.ConnectBusObservers(ctx);
```

Comment:
Well, the bus.
This registers all the bus observers registered in the container - in case of this test, nothing.
Btw @jakubfijalkowski I remember this was introduced to facilitate MT health checks, but it was a long time ago, maybe we should dump it entirely?

Reply:
That's perhaps true. I have a faint memory that it was used for something else but I really can't remember what it was.

We should have a full test that runs real MT w/o any test harnesses. Preferably with ASB or at least RabbitMQ.